src/test/java/edu/indiana/dlib/amppd/service/BatchServiceTests.java

```diff
@@ -264,6 +264,26 @@ public void shouldBeInvalidCollectionName() throws Exception {
         Assert.assertTrue(response.hasErrors());
 	}
 
+	/*
+	 * Deactivated collection.  Should fail validation
+	 */
+	@Test
+	public void shouldBeDeactivatedCollection() throws Exception {
+		String fileName = "batch_manifest_for_testing.csv";
+		ClassLoader classLoader = ClassLoader.getSystemClassLoader();
+
+		File file = new File(classLoader.getResource(fileName).getFile());
+		String content = new String(Files.readAllBytes(file.toPath()));
+
+		System.out.println(content);
+
+		Optional<AmpUser> users = ampUserRepository.findByUsername(ampUsername);
+
+		BatchValidationResponse response = manifestService.validate("Test Unit", "Test File", users.get(), content);
+
+		Assert.assertTrue(response.hasErrors());
+	}
+
```

Comment:
minor suggestion: we try to keep code format consistent. for ex, indent is one tab (or 4 spaces), a blank line is used to separate logic sections, methods, etc (in another word, remove blank lines when not needed). I know not all current code is tidy, but we can try to clean up whenever we see such.